### ISSUE TITLE
Double click edit on a column doesn't work

### ISSUE THREAD

**Issue by phen0menon** [iddan/react-spreadsheet#171]
After updating to the latest release [0.5.16](https://github.com/iddan/react-spreadsheet/releases/tag/v0.5.16) for some reason double click edit stopped working. It just deselect currently selected column on the second click.

Steps to reproduce:

```js
$ git clone https://github.com/iddan/react-spreadsheet
$ yarn
$ yarn storybook
; open localhost:6006 in the browser and check double clicking on a column
```

The problem remains in a project that uses `"react-spreadsheet": "^0.5.16"` package.

**Comment by iddan** [comment]
Publishing a fix in a moment

**Comment by iddan** [comment]
Fixed in `react-spreadsheet@0.5.18`
